src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileSystem.Embedded.targets

```diff
@@ -0,0 +1,61 @@
+<Project>
+
+  <UsingTask
+    AssemblyFile="Microsoft.Extensions.FileSystem.Embedded.Manifest.Task"
+    TaskName="Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.GenerateEmbeddedResourcesManifest" />
+
+  <PropertyGroup>
+    <GenerateEmbeddedFilesManifest Condition="'$(GenerateEmbeddedFilesManifest)' == ''">false</GenerateEmbeddedFilesManifest>
+    <EmbeddedFilesManifestFileName Condition="'$(EmbeddedFilesManifestFileName)' == ''">Microsoft.Extensions.FileSystem.Embedded.Manifest.xml</EmbeddedFilesManifestFileName>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(GenerateEmbeddedFilesManifest)' == 'true'">
+    <EmbeddedResource
+      Include="$(IntermediateOutputPath)$(EmbeddedFilesManifestFileName)"
+      IsManifest="true"
+      Condition="@(EmbeddedResource->WithMetadataValue('IsManifest','true')->Count()) == 0"
```

Comment:
I'm not sure a users would ever need to write their own manifest. Seems better to always let our targets perform the manifest generation.

But if this is an important scenario, you should add a functional test that ensures this scenario works. From what I can tell, the GenerateEmbeddedResourcesManifest task would end up overwriting their file with generated code.